ringpop.go

```diff
@@ -164,25 +172,30 @@ func (rp *Ringpop) identity() (string, error) {
 // TChannel object on the Ringpop instance.
 func (rp *Ringpop) channelIdentityResolver() (string, error) {
 	hostport := rp.channel.PeerInfo().HostPort
+	// Check that TChannel is listening. By default, TChannel listens on an
+	// ephemeral host/port. The real port is then assigned by the OS when
+	// ListenAndService is called. If the hostport is 0.0.0.0:0, it means
+	// TChannel is not yet listening and the hostport cannot be resolved.
+	if hostport == "0.0.0.0:0" {
+		return "", fmt.Errorf("Unable to resolve valid listen address (TChannel hostport is %s)", hostport)
```

Comment:
lower casing error. Normally I settle for lower casing everything except names (eg. `TChannel`), but we should figure out if that is the right way or that it should all be lower cased.